lib/modules/manager/velaci/extract.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { Fixtures } from '../../../../test/fixtures';
+import { extractPackageFile } from './extract';
+
+describe('modules/manager/velaci/extract', () => {
+  describe('extractPackageFile()', () => {
+    it('should handle invalid file', () => {
+      const res = extractPackageFile('\0');
+      expect(res).toBeNull();
+    });
+
+    it('should handle invalid YAML', () => {
```

Comment:
isn't this causing the same as the test above?

Reply:
no, the first throws an error on load, the second gets past the load but returns null for deps. i mean both return null ultimately, but only one covers the try catch for `load`.

Reply:
when the second file loads, then the test is wrongly named. as the yaml parser definitely throws for invalid yaml.

so use an empty file to test the no does case.

Reply:
ok, i reworked/reworded a tests a bit. apparently `foo:bar:invalid\n` is valid but `foo: bar: invalid` is not :) the first should handle invalid yaml in a more apparent manner. the second will test for valid yaml that contains no images (or pipeline).